### PR TITLE
Feature/refactor

### LINKED ISSUE #62: SASS
- [x] Refactor CSS into separate SASS files
- [x] index.scss file for importing partials
- [x] variables.scss containing variables/function definitions

### LINKED ISSUE #63: Accessibility
- [x] Should be able to tab through
- [x] Colorblind accessibility
- [x] Lighthouse score as close to 100% as possible
- [x] ARIA tags only if needed

### PR DESCRIPTION
**Pull request by philalewis**
What (if any) features are you implementing?
- Start using Sass with variables and mixins
- Fix broken tests
- Fix bugs and linting errors
- Style buttons

What (if anything) did you refactor?
- Some CSS
- test files
- Small html changes

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- Please review and merge

Any other comments, questions, or concerns?
- Just need to update the README

### CODE REVIEW

File: src/styles/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@mixin display-flex-row() {
```
Comment (oacortez) [inline src/styles/_mixins.scss:1]: Thank you for creating a mixins file and as well implementing mixins into our regular css file.

File: src/apiCalls.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Fetch requests
 const throwError = (response) => {
-  if(response.ok) {
+  if (response.ok) {
```
Comment (mr-ryan12) [inline src/apiCalls.js:3]: Excellent implementation for the error handling!

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+$nav-color: rgba(0, 128, 128, .5);
```
Comment (oacortez) [inline src/styles/_variables.scss:1]: Thank you for creating a variables scss file and as well implementing them into css file.

File: src/domUpdates.js [hunk 4]
```diff
@@ -10,22 +10,22 @@ const capitalizeFistLetters = string => {
 }
 
 const domUpdates = {
-  updateTags(allTags) {
+  updateTags(allTags, tags) {
```
Comment (mr-ryan12) [inline src/domUpdates.js:13]: Nice catch!

File: src/scripts.js [hunk 5]
```diff
@@ -71,6 +69,9 @@ favoritesButton.addEventListener('click', filterFavorites);
 cookbookButton.addEventListener('click', viewCookbook);
 featuredRecipeImg.addEventListener('click', showRecipeView);
 whatsCookin.addEventListener('click', displayHomeView);
+whatsCookin.addEventListener('keyup', (event) => {
```
Comment (mr-ryan12) [inline src/scripts.js:72]: Excellent solution to make our application more accessible to our users!

File: src/styles/_mixins.scss (hunk 1 above)
Comment (mr-ryan12) [inline src/styles/_mixins.scss:1]: Great use of mixins here!

File: test/Pantry-test.js [hunk 6]
```diff
@@ -36,48 +34,48 @@ describe('Pantry', () => {
       ingredientsData);
     tobasco = new Ingredient(12345, {amount: 1, unit: 'btl'}, ingredientsData);
     bread = new Ingredient(18069, {amount: 2, unit: 'sl'}, ingredientsData);
-    pantry = new Pantry(user.pantry, ingredientsData);
+    user = new User(userData[0], ingredientsData);
```
Comment (mr-ryan12) [inline test/Pantry-test.js:37]: Nice fix to this issue!

**Comment by mr-ryan12** [comment]
Excellent job! Nice work! Next steps will include updating the README.md

**Pull request merged**